conf/svcomp23.json

```diff
@@ -0,0 +1,91 @@
+{
+  "ana": {
+    "sv-comp": {
+      "enabled": true,
+      "functions": true
+    },
+    "int": {
+      "def_exc": true,
+      "enums": false,
+      "interval": true
+    },
+    "activated": [
+      "base",
+      "threadid",
+      "threadflag",
+      "threadreturn",
+      "mallocWrapper",
+      "mutexEvents",
+      "mutex",
+      "access",
+      "race",
+      "escape",
+      "expRelation",
+      "mhp",
+      "assert",
+      "var_eq",
+      "symb_locks",
+      "region",
+      "thread"
+    ],
+    "context": {
+      "widen": false
+    },
+    "malloc": {
+      "wrappers": [
+        "kmalloc",
+        "__kmalloc",
+        "usb_alloc_urb",
+        "__builtin_alloca",
+        "kzalloc",
+
+        "ldv_malloc",
+
+        "kzalloc_node",
+        "ldv_zalloc",
+        "kmalloc_array",
+        "kcalloc",
+
+        "ldv_xmalloc",
+        "ldv_xzalloc",
+        "ldv_calloc"
+      ]
+    },
+    "base": {
+      "arrays": {
+        "domain": "partitioned"
+      }
+    },
+    "autotune": {
+      "enabled": true,
+      "activated": [
+        "singleThreaded",
+        "mallocWrappers",
+        "noRecursiveIntervals",
+        "enums",
+        "arrayDomain",
+        "octagon",
+        "wideningThresholds"
+      ]
+    }
```

Comment:
@leunam99 Is this the set of autotune activations that you'd recommend?

Reply:
Yes it is. One caveat is that this configuration also slightly increased the number of test with a wrong result in my benchmarks. 
(` sv-benchmarks/c/systemc/pipeline.cil-1.yml,`
`sv-benchmarks/c/ldv-linux-3.14-races/linux-3.14--drivers--usb--misc--iowarrior.ko.cil.yml`,
`sv-benchmarks/c/ldv-validator-v0.6/linux-stable-af3071a-1-130_7a-drivers--hwmon--s3c-hwmon.ko-entry_point.cil.out.yml`
and another one I would have to look up later today)

Reply:
- [x] `systemc/pipeline.cil-1` is fixed by https://github.com/goblint/analyzer/pull/859/commits/2c257b60cde8c6b9214f378d92ee82c4f66ba458.
- [x] `ldv-linux-3.14-races/linux-3.14--drivers--usb--misc--iowarrior.ko.cil` is already wrong on master due to a task problem: https://gitlab.com/sosy-lab/benchmarking/sv-benchmarks/-/merge_requests/1374.
- [ ] `ldv-validator-v0.6/linux-stable-af3071a-1-130_7a-drivers--hwmon--s3c-hwmon.ko-entry_point.cil.out` is part of #780.